### PR TITLE
Add scissors config, command & disable command books (1.19)

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: patches/server/0035-Add-Scissors-configuration-file-command.patch [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
++import org.bukkit.command.CommandSender;
++
++import java.io.File;
++import java.util.*;
++import java.util.stream.Collectors;
++
++public class ScissorsCommand extends Command
++{
++
++    private static final String BASE_PERM = "bukkit.command.scissors.";
```
Comment (Telesphoreo) [inline patches/server/0035-Add-Scissors-configuration-file-command.patch:49]: Is there supposed to be a . at the end

File: patches/server/0035-Add-Scissors-configuration-file-command.patch [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
++            This is the main configuration file for Scissors.
++            As you can see, there's tons to configure. Some options may impact gameplay, so use
++            with caution, and make sure you know what each option does before configuring.
++
++            If you need help with the configuration or have any questions related to Scissors,
++            join us in our Discord.
++
++            Discord: https://discord.com/invite/mtVQcHn58h
++            Website: https://scissors.gg/\s
++            Docs: https://scissors.gg/javadoc/1.17.1/\s
```
Comment (Telesphoreo) [inline patches/server/0035-Add-Scissors-configuration-file-command.patch]: Not pointing to the correct version

**Review by Telesphoreo — CHANGES_REQUESTED** [review]

File: patches/server/0035-Add-Scissors-configuration-file-command.patch (hunk 1 above)
Comment (ghost) [reply patches/server/0035-Add-Scissors-configuration-file-command.patch:49]: Yes. This is how Paper does it.

Comment (Telesphoreo) [reply patches/server/0035-Add-Scissors-configuration-file-command.patch:49]: Yep I see it now. I just woke up

**Pull request merged**